rocketpy/Rocket.py

```diff
@@ -579,20 +590,93 @@ def addFins(
         Ymac = (
             (s / 3) * (Cr + 2 * Ct) / Yr
         )  # span wise position of fin's mean aerodynamic chord
+        gamac = np.arctan((Cr - Ct) / (2 * span))
         Lf = np.sqrt((rootChord / 2 - tipChord / 2) ** 2 + span ** 2)
         radius = self.radius if radius == 0 else radius
         d = 2 * radius
+        Aref = np.pi * radius ** 2
+        AR = 2 * s ** 2 / Af  # Barrowman's convention for fin's aspect ratio
         cantAngleRad = np.radians(cantAngle)
-        trapezoidalConstant = ((Yr) / 2) * (radius ** 2) * s
-        trapezoidalConstant += ((Cr + 2 * Ct) / 3) * radius * (s ** 2)
-        trapezoidalConstant += ((Cr + 3 * Ct) / 12) * (s ** 3)
+        trapezoidalConstant = (
+            (Cr + 3 * Ct) * s ** 3
+            + 4 * (Cr + 2 * Ct) * radius * s ** 2
+            + 6 * (Cr + Ct) * s * radius ** 2
+        ) / 12
+
+        # Fin–body interference correction parameters
+        τ = (s + radius) / radius
```

Comment:
not found on barrowman's equations, so I don't know if it's right or not

Reply:
Are you talking about τ or the trapezoildalConstant?
τ is defined in the list of symbols:

![image](https://user-images.githubusercontent.com/69485049/153712358-cb031227-72c7-4ba7-b9d1-33536f66c513.png)

The trapezoildalConstant is everything that is inside the '[ ]' plus 'cr', 's' and '1/6'. So everything that involves the fin's geometry:

![image](https://user-images.githubusercontent.com/69485049/153712702-999f76be-703e-4989-823e-61d5073b9082.png)

In the [roll equations.pdf](https://github.com/MateusStano/RocketPy/blob/f3bbe340fb18d436978745dc5de27e9861c8e8eb/docs/technical/aerodynamics/Roll_Equations.pdf) all this is represented in the integral in equation 20. I did this so it is easier to add more fin shapes in the future and to make the equations easier to understand since on Barrowman it can be quite confusing.

Reply:
I mentioned τ,
well it's clear for me now, tks